lib/localenv/localenv.go

```diff
@@ -71,10 +70,16 @@ type LocalEnvironmentArgs struct {
 	// EtcdRetryTimeout specifies the timeout on ETCD transient errors.
 	// Defaults to EtcdRetryInterval if unspecified
 	EtcdRetryTimeout time.Duration
+	// BoltOpenTimeout specifies the timeout on opening the local state database.
+	// Defaults to defaults.DBOpenTimeout if unspecified
+	BoltOpenTimeout time.Duration
```

Comment:
I added this to support the inherent ambiguity of install/join when looking for operation. This might have a better solution but it works for now. Also, this is not my change but one from [upstream](https://github.com/etcd-io/bbolt/pull/35).